doc/en/user/source/community/index.rst

```diff
@@ -57,5 +57,6 @@ officially part of the GeoServer releases. They are however built along with the
    csw-iso/index
    importer-jdbc/index
    hana/index
-   json-ld/index
+   wfs-templating/index
+   ogc-api/index

```

Comment:
This one is causing a documentation build failure:

```
[exec] /home/travis/build/geoserver/geoserver/doc/en/user/source/community/index.rst:15: WARNING: toctree contains reference to nonexisting document u'community/ogc-api/index'
     [exec] /usr/local/lib/python2.7/dist-packages/sphinx/writers/html.py:462: FutureWarning: 
     [exec]    The iterable returned by Node.traverse()
     [exec]    will become an iterator instead of a list in Docutils > 0.16.
     [exec]   target_node = image_nodes and image_nodes[0] or node.parent
```

Reply:
thanks @aaime fixed